frontend/src/app/main/refs.cljs

```diff
@@ -468,6 +468,14 @@
 (def workspace-active-theme-paths
   (l/derived (d/nilf ctob/get-active-theme-paths) tokens-lib))
 
+(defn token-sets-at-path-all-active?
```

Comment:
This is not a predicate, it returns a derived atom, so it should not be ending with `?`, because it confusing